lib/API/v8/Controller/ModuleController.php

```diff
@@ -115,6 +117,9 @@ public function getModulesMetaList(Request $req, Response $res)
      * @param Request $req
      * @param Response $res
      * @param array $args
+     * @return Response
+     * @throws \SuiteCRM\API\v8\Exception\NotAcceptable
+     * @throws \Psr\Container\ContainerExceptionInterface
      * @throws NotImplementedException
      */
     public function getModulesMetaMenuModules(Request $req, Response $res, array $args)
```

Comment:
parameter type casting here but nowhere else?

Reply:
http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration

Reply:
parameter type casting here but nowhere else. why?

Reply:
The slim framework requires it. Other places do not.

Reply:
I have updated this where necessary.